nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'You must implement the correct method'
+  end
```

Comment:
- [ ] you might want to be a bit more specific with your error, for example, which class and method the error occurs, see below example:
```"#{self.class} has not implemented method '#{__method__}'"```

Reply:
Great suggestion @mikethreels. Will implement it.
And DONE.
Thanks.